datalad/support/gitrepo.py

```diff
@@ -3299,6 +3299,16 @@ def get_content_info(self, paths=None, ref=None, untracked='all',
             paths = [ut.PurePosixPath(p) for p in paths]
 
         path_strs = list(map(str, paths)) if paths else None
+        if path_strs and (not ref or external_versions["cmd:git"] >= "2.27.0"):
```

Comment:
The upstream series didn't graduate to master before v2.28.0-rc0, so the version here and the one in the comment need to be bumped.

Reply:
2.28.0 was released. Should we proceed with this PR?

Reply:
Ah, I misread your comment and thought that it was merged only in that in rc. Got it now